config/database.yml

```diff
@@ -1,25 +1,24 @@
-# SQLite. Versions 3.8.0 and up are supported.
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
-default: &default
-  adapter: sqlite3
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
+# PostgreSQL database
 
 development:
-  <<: *default
-  database: db/development.sqlite3
+  adapter: postgresql
+  encoding: unicode
+  database: blog_db_development
+  pool: 5
+  host: localhost
+  username: blog_admin
+  password: password123
```

Comment:
Bad practice to store credentials in .yml files

Check how rails credentials works:
https://blog.saeloun.com/2019/10/10/rails-6-adds-support-for-multi-environment-credentials.html